website/versioned_docs/version-0.14.0/.old/commands.md

```diff
@@ -0,0 +1,30 @@
+---
+id: commands
+title: CLI commands
+---
+
+## Commands
+
+CLI offers help prompt. 
+Execute `graphback` in your shell for more information
+
+### Config 
+
```

Comment:
I guess it is time to remove the [`/.old`](https://github.com/aerogear/graphback/blob/1231a166c80e3254ff19a0deb4d4040a2cdc1151/website/versioned_docs/version-0.14.0/.old) folder.

Reply:
And the [parent source](https://github.com/aerogear/graphback/tree/1231a166c80e3254ff19a0deb4d4040a2cdc1151/docs/.old) folder too

Reply:
Ouch :D